syncopy/plotting/config.py

```diff
@@ -5,27 +5,51 @@
 
 from syncopy import __plt__
 
+foreground = "#2E3440"  # nord0
+background = '#fcfcfc'  # hint of gray
+
+# dark mode
```

Comment:
Commented out lines should most likely be removed.

Is there a dark mode (planned), btw?